packages/manager/apps/web/client/app/hosting/multisite/hosting-multisite.controller.js

```diff
@@ -129,6 +132,75 @@ angular
               }
             });
           })
+          .then(async () => {
+            $scope.user = await WucUser.getUser();
+          })
+          .then(() => {
+            const getNichandleAttached = async (
+              diagnosticSuffix,
+              domain,
+              nichandle,
+            ) => {
+              const serviceInfos = await HostingDomain.getServiceInfosForZone(
+                domain.name.replace('www.', ''),
```

Comment:
Are you sure that the subdomain couldn't to be other thing that www. for example mysub-domain.my-domain.ovh ?

Reply:
Nope; you're right